app/repositories/users.rb

```diff
@@ -0,0 +1,60 @@
+module Users
```

Comment:
Note: this is an example of the "Repository Pattern" -- a way of separating business logic from data layer logic.

If I have done my job right, the controllers, views, helpers, and process objects will never call the ActiveRecord models directly, but will rather call repository methods.

Reply:
ok. thanks:)